### PR TITLE
{tools}[fosscuda/2019b] Horovod v0.19.5 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node157.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/6680bb78e10caf80405f07471480ce0f for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
taurusa5 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2603 v4 @ 1.70GHz, Python 2.7.5
See https://gist.github.com/f432f9436788aae8e9517dfe172c7e87 for a full test report.

File: easybuild/easyconfigs/h/Horovod/Horovod-0.19.5-fosscuda-2019b-TensorFlow-2.2.0-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+exts_list = [
+    ('cloudpickle', '1.5.0', {
+        'checksums': ['820c9245cebdec7257211cbe88745101d5d6a042bca11336d78ebd4897ddbc82'],
+    }),
+    ('horovod', version, {
+        'checksums': ['428d9ba5ff277467be77e4e707d40b915f7d9e6920a2645f647fcb2cea59c366'],
+    }),
+]
+
```
Comment (lexming) [inline easybuild/easyconfigs/h/Horovod/Horovod-0.19.5-fosscuda-2019b-TensorFlow-2.2.0-Python-3.7.4.eb:35]:
```suggestion

sanity_check_paths = {
    'files': ['bin/horovodrun'],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}

```

**Review by lexming — CHANGES_REQUESTED** [review]
The build works fine. I just have a minor remark. Horovod installs a binary apart from the Python libraries. We should check for it in the sanity checks. Please see below

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node157.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/097bca06f6760285966f58fcffc96e04 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @Flamefire !

**Pull request merged**